runtime/src/test/scala/tailcall/runtime/internal/PrimitiveGen.scala

```diff
@@ -0,0 +1,57 @@
+package tailcall.runtime.internal
+
+import zio.schema.{DynamicValue, StandardType}
+import zio.test.Gen
+
+import java.math.{BigDecimal, BigInteger}
+import java.util.Random
+
+object PrimitiveGen {
+  sealed trait Primitive {
+    type Value
+    val value: Value
+    val standardType: StandardType[Value]
+    val toDynamicValue = DynamicValue.Primitive(value, standardType)
+  }
+
+  final case class Value[A](value: A, standardType: StandardType[A]) extends Primitive {
+    type Value = A
+  }
+
+  private val probablePrime = BigInteger.probablePrime(100, new Random(0x9e3779b1L))
+  private val genBigDecimal = Gen.bigDecimalJava(BigDecimal.ZERO, new BigDecimal(probablePrime))
+  private val genBigInteger = Gen.bigIntegerJava(BigInteger.ZERO, probablePrime)
```

Comment:
Can we use simpler BigInt generator instead of prime, which will just makes test slower.

Reply:
Done.